confignode/src/main/java/org/apache/iotdb/confignode/persistence/ClusterSchemaInfo.java

```diff
@@ -115,17 +103,11 @@ public TSStatus setStorageGroup(SetStorageGroupReq req) {
           .getStorageGroupNodeByStorageGroupPath(partialPathName)
           .setStorageGroupSchema(storageGroupSchema);
 
-      // Mark the new StorageGroup as without Region
-      schemaRegionParticles.add(storageGroupSchema.getName());
-      dataRegionParticles.add(storageGroupSchema.getName());
-
       result.setCode(TSStatusCode.SUCCESS_STATUS.getStatusCode());
-
-      LOGGER.info("Successfully set StorageGroup: {}", storageGroupSchema);
     } catch (MetadataException e) {
```

Comment:
StorageGroupNotExstsEception?

Reply:
I can't modify the type of exception thrown by MTreeAboveSG because it's a complex data structure and not maintained by me...